### ISSUE TITLE
crash over xioamy redmi 13

### ISSUE THREAD

**Issue by traisix** [guardianproject/orbot-android#1057]
**Describe the Bug**

Connection crash frecuenly
Without error, only launcher again
![Screenshot_2024-01-17-17-00-09-696_com android settings](https://github.com/guardianproject/orbot/assets/32341219/3471ca0e-44b8-4375-b0de-25ade5b6b97e)
![Screenshot_2024-01-17-17-08-01-372_org torproject android](https://github.com/guardianproject/orbot/assets/32341219/25bb0eca-acaa-4ce4-9e6c-e28b7f128005)
![Screenshot_2024-01-17-17-07-51-070_org torproject android](https://github.com/guardianproject/orbot/assets/32341219/fcd052d2-e08c-49a1-9667-4ca22b7f5619)
![Screenshot_2024-01-17-17-07-40-311_org torproject android](https://github.com/guardianproject/orbot/assets/32341219/c6ac5436-5245-495e-bf2f-6502bc254d6e)
![Screenshot_2024-01-17-17-07-24-894_org torproject android](https://github.com/guardianproject/orbot/assets/32341219/a6b9d3bb-2bfd-42fc-b714-85b02b414271)
![Screenshot_2024-01-17-17-07-09-347_org torproject android](https://github.com/guardianproject/orbot/assets/32341219/19c900ff-f819-4e15-ad29-eabaeb253ca3)

**What Custom Configuration Do You Use?**
If applicable, how did you set up Orbot? Did you configure some settings?

**Screenshots**

**Smartphone (please complete the following information):**
- Device: [e.g. Pixel 8]
- OS: [e.g. Android 14]
- Version: [e.g. v17.1 RC 2]

**Crash Logs (Advanced)**
If applicable, add crash logs collected using ADB Logcat.

**Additional Context**

**Comment by Decentralizedaz** [comment]
I agree 👍 Latest rc version is constantly crashing more than infected windows95 on any mobile phone used as orbot+Firefox. It always needs to be started again.

**Comment by bitmold** [comment]
There's no crash report here, but there have been a lot of improved stability improvements in recent releases so closing this out of date issue
